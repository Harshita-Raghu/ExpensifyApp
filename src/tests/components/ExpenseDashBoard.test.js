import React from 'react'
import {shallow} from 'enzyme'
import ExpenseDashBoardPage from '../../components/ExpenseDashBoard'

test('should render ExpenseDashBoardPage ',()=>{
    const wrapper=shallow(<ExpenseDashBoardPage/>)
    expect(wrapper).toMatchSnapshot();
})