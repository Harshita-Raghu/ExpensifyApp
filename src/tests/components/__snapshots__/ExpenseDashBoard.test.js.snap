// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashBoardPage  1`] = `
<div>
  This is from dashboard component
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
